bridges/bin/rialto/node/src/chain_spec.rs

```diff
@@ -265,7 +265,7 @@ fn testnet_genesis(
 				// an import since that's a made up number and should be replaced with a constant
 				// obtained by benchmarking anyway.
 				ump_service_total_weight: 4 * 1_000_000_000,
-				max_upward_message_size: 1024 * 1024,
+				max_upward_message_size: 50 * 1024,
```

Comment:
Is there any specific rational for this change?

Reply:
@pepyakin Proposed that value here: https://github.com/paritytech/polkadot/pull/3889#discussion_r758584658